packages/system/reloader/Makefile

```
@@ -1,10 +1,39 @@
+test:
+	helm unittest .
+	# CHART_VERSION names the version that is vendored, and nothing else
+	# enforces that: bumping the pin without re-running `make update` leaves the
+	# committed tree on the old chart and the claim quietly false.
+	@actual_version=$$(awk '/^version:/ {print $$2}' charts/reloader/Chart.yaml | tr -d '"'); \
+	if [ "$$actual_version" != "$(CHART_VERSION)" ]; then \
+	  echo "ERROR: vendored charts/reloader/Chart.yaml version=$$actual_version but Makefile CHART_VERSION=$(CHART_VERSION). Re-run 'make update'."; \
+	  exit 1; \
+	fi
+
+update: clean reloader-update
```

checkmake (0.3.2)

[warning] Target body for "update" exceeds allowed length of 5 lines (8).

(maxbodylength)